File: bloco_4/dia_3/exercicio_2.js
let n = 5;
let result = "";
let asterisk = "*";

for (let i = 1; i <= n; i += 1) {
  for (let j = 0; j < i; j += 1) {
    result += asterisk;
  }
  result += "\n";
}

console.log(result);

File: 4 - CS/Bloco_35/dia_2/exercicios/exercicio_2.py
# Transforme o algoritmo criado do exercicio 1 em recursivo.


def conta_pares(n):
    if n == 1:
        return 0
    if n % 2 == 0:
        return 1 + conta_pares(n - 1)
    else:
        return conta_pares(n - 1)


result = conta_pares(10)
print(result)
